bucket/qview.json

```
@@ -0,0 +1,36 @@
+{
+    "description": "Practical and minimal image viewer",
+    "version": "3.0",
+    "homepage": "https://interversehq.com/qview/",
+    "license": "GPL-3.0",
+    "architecture": {
+        "64bit": {
+            "url": "https://github.com/jurplel/qView/releases/download/3.0/qView-3.0-win64.zip",
+            "hash": "089d07aafab2ec651736903d0a07662210ea93bc34a1b87c3a2a4d6205cb1559"
+        },
+        "32bit": {
+            "url": "https://github.com/jurplel/qView/releases/download/3.0/qView-3.0-win32.zip",
+            "hash": "ffa99c67a096af11816016c94851ef84404be6694a758316da7df2715c48464d"
+        }
+    },
+    "extract_dir": "qView",
+    "shortcuts": [
```

https://github.com/jurplel/qView/blob/25002723c2d4857e2000b9e6cd9d1abb9ea568c9/src/main.cpp#L15-L19

```suggestion
    "bin": "qView.exe",
    "shortcuts": [
```